algebird-core/src/main/scala/com/twitter/algebird/Semigroup.scala

```diff
@@ -195,6 +195,7 @@ object Semigroup extends GeneratedSemigroupImplicits with ProductSemigroups with
   implicit def jshortSemigroup: Semigroup[JShort] = JShortRing
   implicit def longSemigroup: Semigroup[Long] = LongRing
   implicit def bigIntSemigroup: Semigroup[BigInt] = BigIntRing
+  implicit def bigDecimalSemigroup: Semigroup[BigDecimal] = implicitly[Ring[BigDecimal]]
```

Comment:
can we add this for `Monoid` and `Group` too?

Reply:
they're already there... this was the one gap